### PR TITLE
Update issue management with github issues

### LINKED ISSUE #1204: Issue management in pom is still set to jira
### Description
Helix pom still sets jira as the issue management. It should be updated to Apache Helix GitHub issues.
```
   <issueManagement>
    <system>jira</system>
    <url>https://issues.apache.org/jira/browse/HELIX</url>
```

### How to fix
Update issue management in pom with apache helix GitHub  issues.

### PR DESCRIPTION
**Pull request by pkuwm**
### Issues

- [x] My PR addresses the following Helix issues and references them in the PR description:

Fixes #1204 

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

Helix pom still sets jira as the issue management. It should be updated to Apache Helix GitHub issues.

### Tests

- [x] The following tests are written for this issue:

(List the names of added unit/integration tests)

- [x] The following is the result of the "mvn test" command on the appropriate module:

```
mvn site


[INFO] Rendering site with default locale English (en)
[INFO] Relativizing decoration links with respect to localized project URL: http://helix.apache.org/recipes/service-discovery
[INFO] Rendering content with org.apache.maven.skins:maven-default-skin:jar:1.3 skin.
[INFO] Generating "Source Xref" report   --- maven-jxr-plugin:3.0.0:jxr
[INFO] Generating "CI Management" report --- maven-project-info-reports-plugin:3.0.0:ci-management
[INFO] Generating "Dependencies" report  --- maven-project-info-reports-plugin:3.0.0:dependencies
[INFO] Generating "Dependency Information" report --- maven-project-info-reports-plugin:3.0.0:dependency-info
[INFO] Generating "Dependency Management" report --- maven-project-info-reports-plugin:3.0.0:dependency-management
[INFO] Generating "Distribution Management" report --- maven-project-info-reports-plugin:3.0.0:distribution-management
[INFO] Generating "About" report         --- maven-project-info-reports-plugin:3.0.0:index
[INFO] Generating "Issue Management" report --- maven-project-info-reports-plugin:3.0.0:issue-management
[INFO] Generating "Licenses" report      --- maven-project-info-reports-plugin:3.0.0:licenses
[INFO] Generating "Mailing Lists" report --- maven-project-info-reports-plugin:3.0.0:mailing-lists
[INFO] Generating "Plugin Management" report --- maven-project-info-reports-plugin:3.0.0:plugin-management
[INFO] Generating "Plugins" report       --- maven-project-info-reports-plugin:3.0.0:plugins
[INFO] Generating "Source Code Management" report --- maven-project-info-reports-plugin:3.0.0:scm
[INFO] Generating "Summary" report       --- maven-project-info-reports-plugin:3.0.0:summary
[INFO] Generating "Team" report          --- maven-project-info-reports-plugin:3.0.0:team
[INFO] Generating "Sonar" report         --- maven-report:0.1:report
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Apache Helix 1.0.2-SNAPSHOT:
[INFO]
[INFO] Apache Helix ....................................... SUCCESS [ 41.946 s]
[INFO] Apache Helix :: Metrics Common ..................... SUCCESS [  3.780 s]
[INFO] Apache Helix :: Metadata Store Directory Common .... SUCCESS [ 18.297 s]
[INFO] Apache Helix :: ZooKeeper API ...................... SUCCESS [  3.123 s]
[INFO] Apache Helix :: Helix Common ....................... SUCCESS [  1.969 s]
[INFO] Apache Helix :: Core ............................... SUCCESS [  5.277 s]
[INFO] Apache Helix :: Admin Webapp ....................... SUCCESS [  4.270 s]
[INFO] Apache Helix :: Restful Interface .................. SUCCESS [ 13.467 s]
[INFO] Apache Helix :: Distributed Lock ................... SUCCESS [  7.347 s]
[INFO] Apache Helix :: HelixAgent ......................... SUCCESS [  4.432 s]
[INFO] Apache Helix :: Recipes ............................ SUCCESS [  0.389 s]
[INFO] Apache Helix :: Recipes :: Rabbitmq Consumer Group . SUCCESS [  2.572 s]
[INFO] Apache Helix :: Recipes :: Rsync Replicated File Store SUCCESS [  2.354 s]
[INFO] Apache Helix :: Recipes :: distributed lock manager  SUCCESS [  2.080 s]
[INFO] Apache Helix :: Recipes :: distributed task execution SUCCESS [  2.277 s]
[INFO] Apache Helix :: Recipes :: service discovery ....... SUCCESS [  2.160 s]
[INFO] ------------------------------------------------------------------------
diff --git a/pom.xml b/pom.xml
index d613d44c9..a0f63a0c4 100644
--- a/pom.xml
+++ b/pom.xml
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:56 min
[INFO] Finished at: 2020-08-01T21:51:16-07:00
[INFO] ------------------------------------------------------------------------

```

image

### Commits

- [x] My commits all reference appropriate Apache Helix GitHub issues in their subject lines. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation (Optional)

- [ ] In case of new functionality, my PR adds documentation in the following wiki page:

(Link the GitHub wiki you added)

### Code Quality

- [x] My diff has been formatted using helix-style.xml 
(helix-style-intellij.xml if IntelliJ IDE is used)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -367,8 +367,8 @@ under the License.
     <tag>HEAD</tag>
   </scm>
   <issueManagement>
-    <system>jira</system>
-    <url>https://issues.apache.org/jira/browse/HELIX</url>
+    <system>Github</system>
+    <url>https://github.com/apache/helix/issues</url>
```
Comment (narendly) [inline pom.xml:371]: Thanks for making this change.

File: pom.xml [hunk 2]
```diff
@@ -651,14 +651,7 @@ under the License.
...
         <artifactId>maven-site-plugin</artifactId>
-        <version>3.2</version>
-        <dependencies>
-          <dependency>
-            <groupId>org.apache.maven.doxia</groupId>
-            <artifactId>doxia-module-markdown</artifactId>
-            <version>1.3</version>
-          </dependency>
-        </dependencies>
+        <version>3.9.1</version>
```
Comment (narendly) [inline pom.xml:654]: We might need to be a little more careful about making this change - this might mess up the Apache Helix main page...

**Comment by jiajunwang** [comment]
Let's hold on this change before discussing it with the whole Helix committee.

**Comment by jiajunwang** [comment]
@pkuwm Please update here if you have notified the Helix community about this proposal. So I can approve it if no one has concern about the plan.

**Comment by pkuwm** [comment]
@jiajunwang I've sent an email to helix-dev. No one is giving any concern about this. I think it is good to go.

**Review by jiajunwang — APPROVED** [review]

**Comment by pkuwm** [comment]
This PR is ready to be merged, approved by @jiajunwang

**Pull request merged**